CONTRIBUTING.md

```diff
@@ -46,3 +46,7 @@ Run the following command to install the dependencies for the Android SDK:
 In Android Studio, go to `Tools > Android > Sync Project With Gradle Files`.
 
 See the [Gradle Documentation](https://docs.gradle.org/current/userguide/pt02.html) for more information on Gradle and the Gradle Wrapper
+
+## Adding Our Style Guide to Android Studio
+
+We are using spotless to enforce our style guidelines on the project.  You may invoke spotless manually by running `./gradlew check`.  If there are errors spotless will prompt you to run `./gradlew spotlessApply` and auto fix them.  If your code fails to meet the style guidelines it will fail.  However, if you are using Android Studio you may import our style guidelines into your IDE using our [intellij-code-style.xml](codequalite/intellij-code-style.xml) file. 
```

Comment:
Hrm, `./gradlew check` will cause an error because of the debug JSON file problem. 

```
Caused by: java.io.FileNotFoundException at MobileCoreTest.java:42
```

Maybe we should replace it with `./gradlew spotlessCheck`?